app/src/pages/inside/dashboardItemPage/widgetsGrid/emptyWidget/emptyWidget.scss

```diff
@@ -0,0 +1,34 @@
+.empty-widget {
+  width: 100%;
+  padding: 25px 0 60px 0;
+  background-color: $COLOR--white-two;
+}
+
+.empty-widget-headline {
+  margin-top: 27px;
+  margin-bottom: 10px;
+  text-align: center;
+  font-weight: bold;
```

Comment:
Do not use font-weight property ever on our project. If you need bolder font please use existing $FONT-SEMIBOLD constant. Please check all cases.

Reply:
done